backend/MiniLcm/ILexboxApi.cs

```diff
@@ -32,16 +35,24 @@ public interface ILexboxApi
     UpdateBuilder<T> CreateUpdateBuilder<T>() where T : class;
 }
 
-public record QueryOptions(SortOptions Order, int Count = 1000, int Offset = 0)
+public record QueryOptions(SortOptions? Order = null, ExemplarOptions? Exemplar = null, int Count = 1000, int Offset = 0)
 {
-    public static QueryOptions Default { get; } = new(SortOptions.Default);
+    public static QueryOptions Default { get; } = new();
+    public ExemplarOptions? Exemplar { get; init; } = Exemplar ?? ExemplarOptions.Default;
```

Comment:
Looks non-nullable to me
```suggestion
    public ExemplarOptions Exemplar { get; init; } = Exemplar ?? ExemplarOptions.Default;
```